File: C++/2206. Divide Array Into Equal Pairs.cpp
class Solution {
public:
    bool divideArray(vector<int>& n) {
        //sort(n.begin(),n.end());
        int i,fla=0;
        unordered_map<int,int>f;
        for(i=0;i<n.size();i++)
        {
            f[n[i]]++;
        }
        for(i=0;i<f.size();i++)
        {
            if(f[i]%2!=0)
            fla=1;
        }
        return(fla==0);
    }
};

File: JAVA/2206. Divide Array Into Equal Pairs.java
class Solution {
    public boolean divideArray(int[] n) {
        Map<Integer,Integer> m=new HashMap<>();
        for(int i:n){
            m.put(i,m.getOrDefault(i,0)+1);
        }
        for(Integer i:m.values()){
            if(i%2!=0)
            return false;
        }
        return true;
    }
}